enginefs.js

```diff
@@ -201,6 +203,21 @@ router.all("/:infoHash/create", function(req, res) {
         res.end(JSON.stringify(getStatistics(engines[ih])));
     });
 });
+router.all("/create", function(req, res) {
+    var parsed
+    try {
+        var torrent = fs.readFileSync(req.body.from)
```

Comment:
why not async?

also, parse-torrent-file does take a file path, so you don't need `fs` at all. that's why i suggested it